core/src/components/picker-column/picker-column.scss

```diff
@@ -22,17 +57,46 @@
 
   // Hide scrollbars on Firefox
   scrollbar-width: none;
-
-  text-align: center;
 }
 
 /**
  * Hide scrollbars on Chrome and Safari
  */
-:host::-webkit-scrollbar {
+.picker-opts::-webkit-scrollbar {
   display: none;
 }
 
+::slotted([slot="prefix"]), ::slotted([slot="suffix"]) {
+  display: flex;
+
+  flex: 1;
+  align-items: center;
+}
```

Comment:
Currently the slot contains are equal width, so as one slot grows in width so does the other one (even if it doesn't necessarily need it). Just wanted to make sure this is an intended behavior:
 
![image](https://github.com/ionic-team/ionic-framework/assets/2721089/d3f256c4-4e0e-4200-bfb5-7bf7a3218e38)


This behavior seems to align with how the current `ion-picker` works though the difference is the prefix/suffix always evenly take up the remaining white space.

Reply:
Fixed in https://github.com/ionic-team/ionic-framework/pull/28603/commits/cce4c0fabbd907629f7f6384082de8234ff9db9a.